docs/conf.py

```diff
@@ -106,7 +106,8 @@ def pip(*args):
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
-pyproject_toml = toml.load('../pyproject.toml')
+with open('../pyproject.toml', 'rb') as f:
+    pyproject_toml = toml.load(f)
```

Comment:
Should be `tomli`

Reply:
Sorry about that. Will submit a PR in a minute.